styles/index.css

```diff
@@ -2,28 +2,45 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-
-  padding: 60px 20px;
-  gap: 90px;
-  max-width: 800px;
+  background-color: white;
+  padding: 20px 48px;
+  gap: 70px;
+  max-width: 1500;
   margin: 0 auto; /* Center the section in the middle of the page */
 }
 
 .left-corner {
-  flex: 1; /* Allow left section to grow */
+  flex: 1;
+}
+.right-corner {
+  flex: 2;
 }
 
-.profile-pic {
-  /* max-width: 300px; */
-  border-radius: 4px;
-  height: auto;
+.left-corner,
+.readmore-button {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 400; /* Open Sans 400 for regular paragraphs */
+}
+.left-corner h1 {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 600;
 }
 
-.profile-pic:hover {
-  box-shadow: 0 0 2px 1px rgba(53, 55, 55, 0.5);
+.myStory-wrapper {
+  background-color: rgb(253, 238, 240);
```

Comment:
Can we move this color to maybe white instead

Reply:
Yes, lets do that